modules/hcp-aws-tags.adoc

```diff
@@ -0,0 +1,80 @@
+//Module included in the following assemblies:
+// hosted_control_planes/hcp-manage/hcp-manage-aws.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-aws-tags_{context}"]
+= Adding or updating {aws-short} tags for a hosted cluster
+
+As a hosted cluster administrator, you can add or update {aws-first} tags without needing to re-create your hosted cluster. _Tags_ are key-value pairs that are attached to {aws-short} resources for management and automation.
+
+You might want to use tags for the following purposes:
+
+* Managing access controls.
+* Tracking chargeback or showback.
+* Managing cloud IAM conditional permissions.
+* Aggregating resources based on tags. For example, you can query tags to calculate resource usage and billing costs.
+
+You can enable tags to be updated for several different types of resources, including EFS access points, load balancer resources, Amazon EBS volumes, IAM users, and {aws-short} S3.
+
+[IMPORTANT]
+====
+On network load balancers, tags cannot be updated. The {aws-short} load balancer reconciles whatever tags are in the `HostedCluster` resource. If you try to add new tag, the load balancer overwrites everything.
+
+In addition, tags cannot be updated on resources that are created directly by {hcp}. Those resources include EC2 instances, VPC endpoints, and the default security group.
+====
+
+.Prerequisites
+
+* You must have cluster administrator permissions for your hosted cluster on {aws-short}.
+
+.Procedure
+
+. If you are enabling tags for EFS access points, complete the following steps. If you are enabling tags for other types of resources, skip to step 2.
+
+.. In the `aws-efs-csi-driver-operator` service account, add two annotations, as shown in the following example. These annotations are required so that the {aws-short} EKS pod identity webhook that runs on the cluster can correctly assign {aws-short} roles to the pods that the Operator uses.
++
+[source,yaml]
+----
+eks.amazonaws.com/role-arn:<role_arn>
+eks.amazonaws.com/audience:sts.amazonaws.com
+----
+
+.. Delete the operator pod or roll out a restart of the `aws-efs-csi-driver-operator` deployment.
```

Comment:
Capital O Operator pod seems far more common in the docs. 
```suggestion
.. Delete the Operator pod or roll out a restart of the `aws-efs-csi-driver-operator` deployment.
```

Reply:
Fixed